__tests__/unit/utils/sanitizeAndTrimText.test.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  sanitizeText,
+  removeExtraWhiteSpaces,
+  sanitizeAndTrimText,
+} from '../../../app/utils/formInputSanitization';
+
+jest.mock('../../../app/utils/formInputSanitization', () => ({
+  ...jest.requireActual('../../../app/utils/formInputSanitization'), // Use actual implementations where not specifically mocked
+  sanitizeText: jest.fn(),
+  removeExtraWhiteSpaces: jest.fn(),
+}));
+
+describe('sanitizeAndTrimText', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const cases = [
+    {
+      input: 'Hello,   Linkta!@#$   ',
+      sanitized: 'Hello,   Linkta!   ',
```

Comment:
Think this should be `'Hello,   Linkta!      '`, with 6 spaces instead of 3, since the disallowed characters would be replaced with spaces by `sanitizeText`

Reply:
Good catch